index.html

```diff
@@ -21,6 +23,13 @@
         <p>Pat's Salmon Cookies</p>
         <img id="salmon" src="https://github.com/codefellows/seattle-201d8/blob/master/class-09-advanced_css_layout/lab/assets/salmon.png?raw=true">
       </div>
+
+    <section id = "testimonials">
+      <img src="https://github.com/codefellows/seattle-201d8/blob/master/class-09-advanced_css_layout/lab/assets/cutter.jpeg?raw=true">
+
+      <p>Seattle's Original Salmon Cookies</p>
+    </section>
+
     <ul class="bottom_list">
        <li><p class="shop">Shop Swag
```

Comment:
@AlisonZ the markup for this is structured incorrectly - don't see a closing paragraph or parent li tag